code/modules/client/preferences.dm

```diff
@@ -19,6 +19,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 	//game-preferences
 	var/lastchangelog = ""				//Saved changlog filesize to detect if there was a change
 	var/ooccolor = null
+	var/enable_tips = TRUE

```

Comment:
You do not need to bump the version number for added things.

If the thing doesn't exist it defaults to null, and you handle that in loading at the sanitize_ stage of the loading proc, version number bumping is only for things that need to be converted.